deploy/examples/rhoai_kueue.yaml

```diff
@@ -10,15 +10,8 @@ spec:
   config:
     integrations:
       frameworks:
-      - "batch/job"
-      - "kubeflow.org/mpijob"
-      - "ray.io/rayjob"
-      - "ray.io/raycluster"
-      - "jobset.x-k8s.io/jobset"
-      - "kubeflow.org/mxjob"
-      - "kubeflow.org/paddlejob"
-      - "kubeflow.org/pytorchjob"
-      - "kubeflow.org/tfjob"
-      - "kubeflow.org/xgboostjob"
+      - BatchJob
+      - RayJob
+      - RayCluster
       externalFrameworks:
```

Comment:
Let's just remove this from RHOAI example.

This is no longer a valid YAML as appwrappers are moved into frameworks. But RHOAI does not support AppWrappers by default.